src/arrayMethodSort.js

```diff
@@ -3,10 +3,40 @@
 /**
  * Implement method Sort
  */
+const compareByDefault = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(callback = compareByDefault) {
+    let count;
+
+    do {
+      count = 0;
```

Comment:
I suggest to rename it and use as boolean value
```suggestion
      isContinue = false;
```